services/frontend-v3/src/components/officialLanguage/OfficialLanguageView.jsx

```diff
@@ -52,7 +52,7 @@ const OfficialLanguageView = ({ firstLanguageInfo, secondLanguageInfo }) => {
 };
 
 OfficialLanguageView.propTypes = {
-  firstLanguageInfo: PropTypes.isRequired,
+  firstLanguageInfo: PropTypes.array.isRequired,

```

Comment:
You'll need to specify what is in the array, `PropTypes.arrayOf(....).isRequired`

Reply:
on it